modules/apps/user-groups-admin/user-groups-admin-web-test/src/testIntegration/java/com/liferay/user/groups/admin/web/internal/search/test/UserGroupIndexerTest.java

```diff
@@ -91,21 +88,22 @@ public void testSearchUserGroups() throws Exception {
 
 		long companyId = role.getCompanyId();
 
-		int count = userGroupLocalService.searchCount(
+		groupLocalService.addRoleGroup(role.getRoleId(), _group.getGroupId());
+
+		int originalGroupCount = userGroupLocalService.searchCount(
```

Comment:
@julschong Could you rename the variables? They should be about `userGroup*` not `group*`.